docs/book/en/src/dev/python.md

```diff
@@ -0,0 +1,54 @@
+# Python
+
+There are already several different language implementations of the Python runtime, and some of them support WebAssembly. This document will describe how to run [RustPython](https://github.com/RustPython/RustPython) on WasmEdge to execute Python programs.
+
+## Compile RustPython
+
+To compile RustPython, you should have the Rust toolchain installed on your machine. And `wasm32-wasi` platform support is enabled.
+
+```
+$ rustup target add wasm32-wasi
+```
+
+Then you could use the following command to clone and compile RustPython:
+
+```
+$ git clone https://github.com/RustPython/RustPython.git
+$ cd RustPython
+$ cargo build --release --target wasm32-wasi --features="freeze-stdlib"
+```
+
+`freeze-stdlib` feature is enabled for including Python standard library inside the binary file. The output file should be able at `target/wasm32-wasi/release/rustpython.wasm`.
+
+## AOT Compile
+
+WasmEdge supports compiling WebAssembly bytecode programs into native machine code. It is highly recommended to compile the RustPython to native machine code before running.
+
+```
+$ wasmedgec ./target/wasm32-wasi/release/rustpython.wasm ./target/wasm32-wasi/release/rustpython.so
```

Comment:
Since WasmEdge 0.9.0 supported universal binary, we could use `wasmedgec xyz.wasm xyz.wasm` to generate the native binary.  For more details, please refer to the release article - Universal Wasm Binary part. https://www.secondstate.io/articles/wasmedge-0.9.0-release/